src/app/pages/SmartEventsPage/SmartEventsPage.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { FederatedModule } from "@app/components";
+import { AppServicesLoading } from "@rhoas/app-services-ui-components";
+import { useAuth, useBasename, useConfig } from "@rhoas/app-services-ui-shared";
+
+const SmartEventsPage: React.FC = () => {
+  const auth = useAuth();
+  const config = useConfig();
+  const basename = useBasename();
+
+  return (
+    <>
+      <FederatedModule
+        scope="smart_events"
+        fallback={<AppServicesLoading />}
+        module="./SmartEventsApp"
+        render={(SmartEventsApp) => (
+          <SmartEventsApp
```

Comment:
I'd suggest creating a `SmartEventsPageConnected` component where you consume the hooks to avoid re-fetching the federated module needlessly on auth/config/basename changes. It shouldn't happen but the current implementation of those hooks doesn't guarantee that.